lib/puma/server.rb

```diff
@@ -330,9 +331,10 @@ def handle_servers
           [nil, nil]
         end
 
-        while @status == :run || (drain && shutting_down?)
+        shd = false
```

Comment:
```suggestion
        is_shutting_down = false
```

I think we should avoid cryptic abbreviations

Reply:
100% agree with you 😄 
Fixed!